docs/release_notes/8_6.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Release 8.6
+---
+
+# Release Notes For Rocky Linux 8.6
+
+##  Major Changes
+
+For a complete list of major changes, please see the upstream listing [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.6_release_notes/overview#overview-major-changes).
+
+Below are some highlights.
+
+### Security
+
+ * `fapolicyd`: updated to version 1.1. Includes the ability to use `/rules.d` and `/trust.d` directories, the ability to use `fagenrules` script, and new options for the `fapolicyd-cli`command.
+ * OpenSSH server now supports drop in configurations. This means you can use the `Include` directive in `sshd_config` similar to what is now available for `ssh_config`.
+ * `pcsc-lite` package: rebased to upstream version 1.9.5 with new enhancements and bug fixes.
+ * SELinux: You can now verify versions of installed policy modules by using the `--checksum` option with the `semodule` command.
+ *  SCAP Security Guide (SSG) packages:  rebased to version 0.1.60, and OpenSCAP packages have been rebased to version 1.3.6.
+
+### Web and Programming
+
+* PHP 8.0
+* Perl 5.32
+
+### Identity Management
+
+* `ansible-freeipa` roles and modules available in the Ansible Automation Hub.
+
```

Comment:
Add a section for "Infrastructure Services" and mention that `bind9.16` was introduced as an alternative to the regular `bind` package.

Reply:
> Add a section for "Infrastructure Services" and mention that `bind9.16` was introduced as an alternative to the regular `bind` package.

Will do!